src/TLuaInterpreter.cpp

```diff
@@ -568,7 +568,9 @@ void TLuaInterpreter::handleHttpOK(QNetworkReply* reply)
             break;
         }
 
-        localFile.flush();
+        if (!localFile.commit()) {
```

Comment:
:thinking: Presumably - if the `QSaveFile::commit()` has failed then automatically the `QSaveFile::error()` value will NOT be **`QFileDevice::NoError`** (and I guess `QSaveFile::NoError` == `QFile::NoError` == `QFileDevice::NoError`)?

Reply:
Right.